scripts/populate_institutions.py

```diff
@@ -170,17 +180,19 @@ def main(env):
                 'logout_url': SHIBBOLETH_SP_LOGOUT.format(encode_uri_component('https://shibboleth.nyu.edu/idp/profile/Logout')),
                 'domains': ['osf.nyu.edu'],
                 'email_domains': [],
+                'delegation_protocol': 'saml-shib',
             },
             {
                 '_id': 'okstate',
                 'name': 'Oklahoma State University',
                 'description': '<a href="http://www.library.okstate.edu/research-support/research-data-services/">OSU Library Research Data Services</a>',
                 'banner_name': 'okstate-banner.png',
                 'logo_name': 'okstate-shield.png',
-                'login_url': None,  # https://stwcas.okstate.edu/cas/login?service=...
-                'logout_url': None,
+                'login_url': None,
```

Comment:
Before: `login_url` is used by CAS to decide which institutions show up on the login list.
After: `delegation_protocol` is used and `login_url` is `None` for institutions that are delegated through external CAS IdP.